FlightStreamDeck.AddOn/deploy.bat

```diff
@@ -1,2 +1,3 @@
 taskkill -f -t -im StreamDeck.exe -fi "status eq running"
-XCOPY "." "%appdata%\Elgato\StreamDeck\Plugins\tech.flighttracker.streamdeck.sdPlugin\" /S /E /Y
\ No newline at end of file
+XCOPY "." "%appdata%\Elgato\StreamDeck\Plugins\tech.flighttracker.streamdeck.sdPlugin\" /S /E /Y
+START /d "%ProgramW6432%\Elgato\StreamDeck" StreamDeck.exe
```

Comment:
Do you mind reverting this change for this PR? I'll add it directly in the next commit.

Reply:
updated.